model-developers/scripts/compile-circuit.ts

```diff
@@ -0,0 +1,12 @@
+#! ./node_modules/.bin/ts-node
+
+import 'zx/globals';
+
+const CIRCUIT_NAME: string = argv.CIRCUIT_NAME || "demo-circuit";
```

Comment:
Generalized these commands, so that any model should be able to compiled just by passing argument(s) to them.